xs/xsp/Print.xsp

```diff
@@ -178,9 +178,14 @@ _constant()
     // TODO: status_cb
     Ref<ExtrusionEntityCollection> skirt()
         %code%{ RETVAL = &THIS->skirt; %};
-    Ref<ExtrusionEntityCollection> brim()
+    Ref<std::vector<ExtrusionEntityCollection> > brim()
```

Comment:
this is probably useless now, but I am very bad at using XS to get a useful kind of reference.